core/controllers/acl_decorators_test.py

```
@@ -47,18 +48,24 @@
     user_services,
 )
 from core.platform import models
+from core.storage.certificate_assessment import (
+    gae_models as certificate_assessment_models,
+)
 from core.tests import test_utils
 
 import webapp2
 import webtest
-from typing import Any, Dict, Final, List, Union
+from typing import Any, Dict, Final, List, Optional, Union
```

Ruff (0.16.1)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)